app/src/shared/components/CalendarIcon/calendarIcon.pcss

```diff
@@ -1,8 +1,10 @@
 .root {
```

Comment:
`CalendarIcon` may look different on different breakpoints because of `rem`. Example:

Tablet:

<img width="605" alt="Screenshot 2019-11-26 22 29 30" src="https://user-images.githubusercontent.com/320066/69674302-7e696380-109c-11ea-940e-f40939d51935.png">

Desktop:

<img width="748" alt="Screenshot 2019-11-26 22 31 02" src="https://user-images.githubusercontent.com/320066/69674290-74476500-109c-11ea-91ab-64d042807ec4.png">

Looks like text field and buttons have the same height on all resolutions. Maybe the same for the calendar icon?